### PR TITLE
Livia Chart 2

### PR DESCRIPTION
**Pull request by ChuyueSun**

### CODE REVIEW

**Comment by ChuyueSun** [comment]
Screen Shot 2019-04-12 at 12 57 51 AM
Screen Shot 2019-04-12 at 12 58 22 AM

File: src/main/webapp/chart.html [hunk 1]
```diff
@@ -3,30 +3,51 @@
...
+            var msgData = new google.visualization.DataTable();
+            //define columns for the DataTable instance
+            msgData.addColumn('date', 'Date');
+            msgData.addColumn('number', 'Message Count');
+            // console.log(msgJson);
+
+            for (i = 0; i < msgJson.length; i++) {
+                msgRow = [];
+                var timestampAsDate = new Date (msgJson[i].timestamp);
+                // var newDate = new Date(timestampAsDate.getYear(), timestampAsDate.getMonth(), timestampAsDate.getDay());
```
Comment (tsalako) [inline src/main/webapp/chart.html]: we dont need this comment.

**Review by tsalako — CHANGES_REQUESTED** [review]
Can you add a screenshot of the chart?

File: src/main/webapp/chart.html (hunk 1 above)
Comment (ChuyueSun) [reply src/main/webapp/chart.html]: What do you mean?

**Review by ChuyueSun — COMMENTED** [review]
Screen Shot 2019-04-16 at 3 17 52 PM
This is what it currently looks like...

**Comment by tsalako** [comment]
The chart doesnt seem to be working. Can you look into fixing it/see where it is breaking down.

File: src/main/webapp/chart.html (hunk 1 above)
Comment (tsalako) [reply src/main/webapp/chart.html]: newDate isnt commented out and we dont need it so you can remove the comment.

**Pull request closed without merging**